pages/linux/create_ap.md

```diff
@@ -0,0 +1,35 @@
+# create_ap
+
+> Create an AP (Access Point) at any channel.
+
+- No passphrase (open network):
+
+`create_ap wlan0 eth0 MyAccessPoint`
+
+- For an example, I have internet access on my eth0 network, and my wireless interface is wlan0, I want the name of my network to be “ubuntuGuide” and the password as “12345678” so:
+
+`create_ap eth0 wlan0 ubuntuGuide 12345678`
```

Comment:
In cases where the user inputs a parameter that is not predefined by the command we use curly brackets around that parameter.

`create_ap eth0 wlan0 {{ubuntuGuide}} {{12345678}}`

You can have a better explanation by reading the [maintainers guide](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/maintainers-guide.md) and the [style guides](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/style-guide.md).